mlflow/pytorch/_pytorch_autolog.py

```diff
@@ -29,7 +30,20 @@
 
 
 @rank_zero_only
+@experimental
 def _autolog(log_every_n_epoch=1):
+    """
+    Enable automatic logging from pytorch to MLflow.
```

Comment:
used the doc string from [fastai autolog](https://github.com/mlflow/mlflow/blob/89fb38ec122a1c489488ab1fb0d35fe70c2be649/mlflow/fastai.py#L282) method with minor changes.